common/armies/hfe_armies.txt

```diff
@@ -38,3 +38,303 @@
 # 		}
 # 	}
 # }
+
+hfe_army_high_mechanicus_bodyguards = {
+	defensive = yes
+	is_pop_spawned = no
+	pop_limited = no
+	health = 1.1
+	damage = 2
+	morale = 5
+	has_morale = yes
+	morale_damage = 1.5
+	collateral_damage = 0.9
+	war_exhaustion = 2
+	icon_frame = 5
+	has_species = yes
+
+	resources = {
+		category = armies
+		cost = {
+			trigger = {
+				owner = {
+					NOT = {
+						is_primitive = yes
+						is_country_type = primitive
```

Comment:
The way I set it up, the Varelse can rebuild one of their special Armies. The reason that is there is so that Primitive Armies are deleted when the Varelse level up.

Reply:
Still, make no sense to add primitive to cost trigger

Reply:
The reason I did that was to charge Primitives a different amount from a FTL Varelse country.

Reply:
Primitive have no economy
And ai
They will not hire armies and will not pay upkeep

Reply:
I have mentioned this over and over again. The Varelse keep using their special Armies no matter what Primitive Age they are on, and that includes when they achieve FTL.